src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -537,29 +538,25 @@ public bool IsSelectedItemShortcut
             set => SetProperty(ref isSelectedItemShortcut, value);
         }
 
-        public void CheckFileExtension(string itemExtension)
+        public void CheckAllFileExtensions(List<string> itemExtensions)
         {
-            // Set properties to false
-            IsSelectedItemImage = false;
-            IsSelectedItemShortcut = false;
+            // Checks if all the item extensions are image extensions of some kind.
+            IsSelectedItemImage = itemExtensions.TrueForAll(itemExtension => this.IsImageExtension(itemExtension));
+            // Checks if there is only one selected item and if it's a shortcut.
+            IsSelectedItemShortcut = (itemExtensions.Count == 1) && (itemExtensions.TrueForAll(itemExtension => this.IsShortcutExtension(itemExtension)));
```

Comment:
Might be worthwhile moving the `IsShortcutExtension` method into the `FileExtensionHelpers` class as well?

Reply:
Done!